cmd/stack_package.go

```diff
@@ -192,8 +304,13 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			// overriding time label with stack package currentTime generated earlier
+			labels["org.opencontainers.image.created"] = currentTime
```

Comment:
You should use `labels[ociKeyPrefix+"created"]` here.

There's actually quite a lot of overlap in the labels and the template variables. I'd like you to think about how to reuse more of the code. Perhaps you can get the labels first and then get the template variables from the labels? Or maybe encapsulate all the stack metadata into a single object/type and have methods to get label maps or template maps.

Reply:
Also I really like this better. Thanks!